Gems/LyShine/Code/Tests/TextInputComponentTest.cpp

```diff
@@ -96,14 +96,14 @@ namespace UnitTest
     };
 
     class UiTextInputComponentTest
-        : public ::testing::Test
+        : public UnitTest::AllocatorsTestFixture
```

Comment:
repeat: This can be changed to inherit from ScopedAllocatorSetupFixture and the `AllocatorInstance<SystemAllocator>` Create and Destroy calls can be removed.